### DISCUSSION TITLE
Need help to get best practice to control 2 incompatible BMS Seplos (V2 & V3) on same victron

### DISCUSSION THREAD

**Discussion by zigomatichub** [mr-manuel/venus-os_dbus-serialbattery#132] [Q&A] [answered]
Seplos V2 (positive)  and V3 (negative) BMS cannot be used together in // mode. 
Then I plan to have 2 connections RS485. 
The question are : 

- dbus-serial is able to cover 2 seplos instance ?  
  quid of the parameters related to BMS V3 when there are 2 different BMS version for the same vendor ?

- Could have 2 dbus-serial instance in //, like 2 differents battery BMS running on the same victron ? But then something need to aggregate the data to push to victron the most limiting values ?

- Worst case scenario only possible : control only one of them ?

**Comment by mr-manuel** [comment, marked as answer]
All your questions should be answered in the documentation.

**Reply by zigomatichub** [reply]
Oh my bad. I was thinking I'm in a special case.

I'm sorry for that, I keep open and look into documentation and reply/close when it's done. 

Thank you for the message Mr mr-manuel :)
